jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-showcase/README.md

```diff
@@ -79,4 +79,35 @@ Start the server by running:
 
 Verify the application is running. Go to http://SERVER:PORT/jbpm-cm/ and type the specified username and password. You should see simple XML message with basic information about the server.
 
-For more details, please check the [Getting Started Guide](https://docs.jboss.org/author/display/WFLY10/Getting+Started+Guide).
\ No newline at end of file
+For more details, please check the [Getting Started Guide](https://docs.jboss.org/author/display/WFLY10/Getting+Started+Guide).
+
+## Demo Docker image
+
+To allow you to quickly get started with the jBPM Case Management Showcase fully integrated with a running Kie Server, a Docker image can be generated as part of the build.
+This image contains both applications deployed to a single WildFly 10 instance. It also provides the settings necessary for showcase to retrieve the necessary data from the running Kie Server instance.
+By default, a few demo containers are deployed to allow you to interact with the Case Management concepts as well as pre-defined users with the following credentials: admin/admin and user/user. 
```

Comment:
@cristianonicolai I tried to build & run the docker image locally.
I managed to start it successfully (although I needed to add `sudo` to all maven commands that have `-Ddocker`)

The problem is that I don't see any sample containers deployed. I'm not sure what you mean by "By default, a few demo containers are deployed".
Is this what you intend to implement by setting `-Dorg.kie.demo=true`?
If I understand it correctly this property was only used in the past in the kie-wb's `AppSetup.java` (but on master it is no longer used there) but we are not even deploying kie-wb in this scenario. So this does nothing for the setup of demo containers on kie-server.

Any idea how to get some case definitions in this setup without having to interact with kie-server rest api directly? :-)